apps/internal/local/server.go

```diff
@@ -145,7 +153,7 @@ func (s *Server) handler(w http.ResponseWriter, r *http.Request) {
 		desc := html.EscapeString(q.Get("error_description"))
 		// Note: It is a little weird we handle some errors by not going to the failPage. If they all should,
 		// change this to s.error() and make s.error() write the failPage instead of an error code.
-		_, _ = w.Write([]byte(fmt.Sprintf(failPage, headerErr, desc)))
+		_, _ = w.Write([]byte(fmt.Sprintf(string(failPage), headerErr, desc)))
```

Comment:
This tries to interpolate a couple strings into `failPage`, which may not contain format verbs. I guess someone could learn that by reading this code, or we could document it somewhere, but it raises the question: is this interpolation part of the API contract?

Reply:
@chlowell Thanks for checking this. I have fixed this, by removing the string interpolation, and also moved the scope of the success and error page to the lifetime of the server object.